app/build.gradle

```diff
@@ -314,10 +314,10 @@ dependencies {
     implementation 'com.google.firebase:firebase-crashlytics'
     implementation 'com.google.firebase:firebase-crashlytics-ndk'
     implementation('com.google.firebase:firebase-iid:21.1.0')
+    implementation('com.google.firebase:firebase-messaging')
```

Comment:
@josikie  Lets add the In-App messaging dep:
`implementation("com.google.firebase:firebase-inappmessaging-display")`